src/sota_tools/garage_check.cc

```diff
@@ -160,6 +161,22 @@ int main(int argc, char **argv) {
   }
 
   Json::Value targets_json = Utils::parseJSON(targets_str);
+  std::string expiry_time_str = targets_json["signed"]["expires"].asString();
+  if (expiry_time_str.length() != 20 || expiry_time_str[19] != 'Z') {
+    LOG_FATAL << "'expires' field has wrong format";
+    return EXIT_FAILURE;
+  }
+
+  struct tm expiry_time {};
+  strptime(expiry_time_str.c_str(), "%Y-%m-%dT%H:%M:%SZ", &expiry_time);
```

Comment:
We already have code to parse timestamps in `[Uptane::TimeStamp](https://github.com/advancedtelematic/aktualizr/blob/b2705bd25d3ccc9eda10421abd6a0a45125adf7b/src/libaktualizr/uptane/tuf.h#L63)`, and we should use it here--this will eliminate duplicating the implementation, and will also mean that we are checking that the timestamp is valid using exactly the same code that will we used later on when Aktualizr validates it.